godump.go

```
@@ -118,6 +119,24 @@ func DumpHTML(vs ...any) string {
 	return sb.String()
 }
 
+// DumpJSON dumps the values as a pretty-printed JSON string.
+// If there is more than one value, they are dumped as a JSON array.
+// It returns an error string if marshalling fails.
+func DumpJSON(vs ...any) string {
+	var data any
+	if len(vs) == 1 {
+		data = vs[0]
+	} else {
+		data = vs
+	}
```

I would return an error if the variadic is empty (len() = 0)